proto/translate/v1/translate.proto

```diff
@@ -151,7 +152,7 @@ service TranslateService {
 
   rpc UpdateMessages(UpdateMessagesRequest) returns (Messages) {
     option (google.api.http) = {
-      put: "/v1/services/{service_id}/messages"
+      put: "/v1/services/{service_id}/messages/{language}"
```

Comment:
Does proto allow the following?

```suggestion
      put: "/v1/services/{service_id}/messages/{messages.language}"
```

Reply:
It is possible but might cause issues when making requests using REST client. By providing different language in the path  messages.language will be overwritten.

Reply:
I see. The issue is an incorrectly constructed REST request (bad request) processed successfully when the duplicate field is not equal. On the opposite side, it degrades the gRPC request by adding the duplicate field. In our case, gRPC is the primary interface, we should keep it clean. Let's use `{messages.language}`! REST API is nice to have anyway.